app/src/main/java/com/pvp/app/api/PointService.kt

```diff
@@ -32,6 +33,16 @@ interface PointService {
         return tasks.sumOf { calculate(it) }
     }
 
+    /**
+     * Calculate the points of a goal. Points are determined by the goal type and the attributes of
+     * the goal. User status can also be taken into account, hence, [Goal.email] is required.
```

Comment:
_can be taken into account_ / _is required_, those are two opposites, which is it then? 😄
Also, if you call it required, then an error would be expected when email is not provided.

Reply:
The comment is just copied comment from `calculate` method for tasks, since it basically does the same. In that case, both need to be changed.